app/models/game.rb

```diff
@@ -57,4 +57,13 @@ def player_color(player)
     return 'black' if player.id == black_player_id
     false
   end
+
+  # keep track of whose turn
+  def whose_turn(color)
+    if color
```

Comment:
Both white and black will be "truthy" here since they are strings, so you probably want to make this explicit, like `if color == 'black'`.  Then if it's called with `black` it will set the turn to white and vice versa.

Also, something probably needs to call this method - in the piece controller update action, you could call this right after a move succeeded, to change the turn to the other player.